dhall/src/Dhall/Main.hs

```diff
@@ -205,6 +208,13 @@ parseOptions =
             <>  Options.Applicative.help description
             )
 
+    parseCommentControl = fmap f $ switch
+        "loose-comment-parsing"
+        "Ignore parse error for unhandled comments, but they will get dropped (please submit an issue for such cases)"
```

Comment:
```suggestion
        "discard-unsupported-comments"
        "Discard comments that cannot be auto-formatted instead of failing (but continue to submit issues for any comments the auto-formatter rejects)"
```

Reply:
Nice! Much better wording :+1: